spec/models/form_profile_spec.rb

```diff
@@ -8,26 +8,22 @@
 RSpec.describe FormProfile, type: :model do
```

Comment:
Removed Contact Info V1 from specs! 🎉 form_profile_v2_spec code was migrated to form_profile_spec